### PR TITLE
Add exclude to cmd

### PR DESCRIPTION
**Pull request by itdove**
Signed-off-by: Dominique Vernier <dvernier@redhat.com>

### CODE REVIEW

File: pkg/templateprocessor/yamlstringreader_test.go [hunk 1]
```diff
@@ -20,6 +20,7 @@ subjects:
   name: {{ .BootstrapServiceAccountName }}
   namespace: {{ .ManagedClusterNamespace }}`, `apiVersion: v1
 kind: ServiceAccount
+# hello ---
```
Comment (hanqiuzh) [inline pkg/templateprocessor/yamlstringreader_test.go:23]: hello?

Comment (hanqiuzh) [reply pkg/templateprocessor/yamlstringreader_test.go:23]: I guess this is to test non starting delimiter?

Comment (itdove) [reply pkg/templateprocessor/yamlstringreader_test.go:23]: yes in test.... a comment

Comment (itdove) [reply pkg/templateprocessor/yamlstringreader_test.go:23]: you want something more pro ?

Comment (hanqiuzh) [reply pkg/templateprocessor/yamlstringreader_test.go:23]: 😂  no, I think `hello` is good 👍

**Pull request merged**